frontend/packages/console-app/src/components/cloud-shell/CloudShellTerminal.tsx

```diff
@@ -223,7 +223,7 @@ const CloudShellTerminal: React.FC<CloudShellTerminalProps &
 export const InternalCloudShellTerminal = CloudShellTerminal;
 
 const stateToProps = (state: RootState): StateProps => ({
-  user: state.UI.get('user'),
+  user: state.core.user,
```

Comment:
I think we need to use selectors the SDK should provide here... going directly into our reducer structure is asking for trouble for apps. If we want properties to change, we'll never be able to do it if everyone writes their own walk through our state.

[Something like this](https://www.typescriptlang.org/play?#code/PTAEGUBEGkIFwIZwKagCoE8AOyDOBYAKDm1QGEB7AJ2QCVkATAVwGNkrxEVQBeUAbyKhQTXOwBcoXHCoBLAHYBzANxEAvqsJESOCDE7VUfQYWEtDkyjXrM2HLsk0aiLwiFABBLFgA2slkiyFPLwSKg6yNqknt4GNLwCQqAUcAAWEqDyTAC2AEbsTqAAZHrQcY6uRO4AogAeWBRiDKUQyD7ILHDUROby0lJtHV1UAGLUAKpiVAkAFNJhklBlw8gAlJaGNqzsnGEA2gDkouwHALq8AHyJpqA0cExUIfMoAHTmNC-HVE6arjW1CGyvlQogQikihF6-V68iG5V23GMyTSGQArAAaUDvZCSfgiKaSA4AKwYFGQB1AakpmihcHx7ASYnanWoYyok3YMxhcJWCLWmiAA)

Reply:
yeah make sense, I just update the current state with new key access. Have made use of selectors which have already been exposed.